semiphemeral/twitter.py

```diff
@@ -794,14 +794,14 @@ def import_dump(self, filepath):
 
             if os.path.isdir(filepath):
                 # Unzipped tweet archive
-                with open(os.path.join(filepath, "data", "tweet.js"), "r", encoding="UTF-8") as f:
+                with open(os.path.join(filepath, "data", "tweets.js"), "r", encoding="UTF-8") as f:
                     # Skip the JS variable assignment at the start of this file
                     f.read(25)
                     tweets = json.load(f)
             elif os.path.splitext(filepath)[1] == ".zip":
                 # Zipped tweet archive
                 with ZipFile(filepath) as zipfile:
-                    with zipfile.open("data/tweet.js") as f:
+                    with zipfile.open("data/tweets.js") as f:
                         f = io.TextIOWrapper(f, "UTF-8")
                         # Skip the JS variable assignment at the start of this file
                         f.read(25)
```

Comment:
For anyone wondering: this part of the code continues to work even with the rename because of a space character:
```js
// before, with tweet.js
window.YTD.tweet.part0 = [
//                      ^ 25th char
// after, with tweets.js
window.YTD.tweets.part0 = [
//                      ^ 25th char
```